### PR TITLE
Split benchmark.yml

### PR DESCRIPTION
**Pull request by jasdom**
`benchmark.yml` needs to be split into separate workflows to store test results separately.

### CODE REVIEW

File: .github/workflows/benchmark-hierarchies.yml [hunk 1]
```diff
@@ -7,15 +7,15 @@ on:
       - master
     paths:
       - apps/performance-tests/**
```
Comment (grigasp) [inline .github/workflows/benchmark-hierarchies.yml:9]: should this point to `hierarchies` sub-directory?

File: .github/workflows/benchmark-unified-selection.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
+name: Unified-selection performance benchmark
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    paths:
+      - apps/performance-tests/**
```
Comment (grigasp) [inline .github/workflows/benchmark-unified-selection.yml:9]: shouldn't this be removed?

File: .github/workflows/benchmark-hierarchies.yml (hunk 1 above)
Comment (jasdom) [reply .github/workflows/benchmark-hierarchies.yml]: Yes, it should. Fixed.

File: .github/workflows/benchmark-unified-selection.yml (hunk 2 above)
Comment (jasdom) [reply .github/workflows/benchmark-unified-selection.yml]: Removed.

**Review by grigasp — APPROVED** [review]

**Pull request merged**